community/wavemon/APKBUILD

```diff
@@ -14,37 +14,33 @@ source="$pkgname-$pkgver.tar.gz::https://github.com/uoaerg/$pkgname/archive/v$pk
 builddir="$srcdir/$pkgname-$pkgver"
 
 prepare() {
-	default_prepare || return 1
```

Comment:
This shouldn't be removed, otherwise patches aren't applied.

Reply:
Sure. Fixed.